examples/treeview/css/treeview-navigation.css

```diff
@@ -0,0 +1,156 @@
+#ex1 header {
```

Comment:
This is why the CSS isn't applying to the example when it is opened in codepen! Can you use something else of the CSS specifiers? Same reason as here: https://github.com/w3c/aria-practices/pull/1612/files#r535718223

Reply:
fixed